docker-compose.yaml

```diff
@@ -27,6 +27,7 @@ services:
             - NANGO_DB_SSL=${NANGO_DB_SSL}
             - NANGO_DB_POOL_MIN=${NANGO_DB_POOL_MIN}
             - NANGO_DB_POOL_MAX=${NANGO_DB_POOL_MAX}
+            - RECORDS_DATABASE_URL=postgres://${NANGO_DB_USER}:${NANGO_DB_PASSWORD}@${NANGO_DB_HOST}:5432/${NANGO_DB_NAME}$([[ "${NANGO_DB_SSL}" == "true" ]] && echo "?sslmode=require")
```

Comment:
Could have been easier in the code. Also it's not documented in .env.example

Reply:
I didn't document it because it it is all based on other env vars that are documented. Also as mentioned I am not sure it is necessary since hosted is only used for oauth